### PR TITLE
Release notes v1.53

### PR DESCRIPTION
**Pull request by josunect**
Release notes for version v. 1.53
https://github.com/orgs/kiali/projects/59/

They have been generated using the script proposed in https://github.com/kiali/kiali.io/pull/558 

https://deploy-preview-560--kiali.netlify.app/news/release-notes/

### CODE REVIEW

**Comment by israel-hdez** [comment]
@jmazzitelli would you like to review? I think you are more used to what (and what not) should be included in release notes.

File: content/en/news/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,37 @@ weight: 1
...
+* [Add outboundTrafficPolicy value to overview istio-system card ](https://github.com/kiali/kiali/issues/4896)
+* [Create a UI suite test on cypress](https://github.com/kiali/kiali/issues/4854)
+* [UI tests around the Workloads Details page.](https://github.com/kiali/kiali/issues/5072)
+* [Add Kiali validations on the Istio Config Details sidepanel](https://github.com/kiali/kiali/issues/5204)
+* [Not found messages may have a better message in the body page](https://github.com/kiali/kiali/issues/5206)
+* [Reproducible performance testing environment](https://github.com/kiali/kiali/issues/5142)
+* [Envoy tab: add tooltips with the Envoy terminology](https://github.com/kiali/kiali/issues/5162)
+* [Update the information about mTLS data in details pages](https://github.com/kiali/kiali/issues/5187)
+* [Document ability to set default Kiali CR image_name in operator from helm charts](https://github.com/kiali/kiali/issues/5238)
+* [(hack) how to reduce cpu/memory resources consumed by some istio components](https://github.com/kiali/kiali/issues/5264)
```
Comment (jmazzitelli) [inline content/en/news/release-notes.md]:
```suggestion
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
This looks good! thanks @josunect 

The only recommended change is to remove the hack PR -- we typically don't announce changes we make to the hack scripts since those are only relevant to kiali developers.

**Comment by josunect** [comment]
Thanks, @jmazzitelli ! Proposed change has been committed.

**Review by jmazzitelli — APPROVED** [review]
lgtm -- we'll fix the htmlproofer error in another pr

**Comment by israel-hdez** [comment]
@jmazzitelli @josunect I think this needs a backport into `current`, no?

**Comment by jmazzitelli** [comment]
> @jmazzitelli @josunect I think this needs a backport into `current`, no?

ah, yes.. forgot about that. Right - this went into "staging" but we also need it in "current" because it needs to go in the current website content.

@josunect just create a cherry-pick PR that puts this commit over in the "current" branch.

**Comment by josunect** [comment]
> > @jmazzitelli @josunect I think this needs a backport into `current`, no?
> 
> ah, yes.. forgot about that. Right - this went into "staging" but we also need it in "current" because it needs to go in the current website content.
> 
> @josunect just create a cherry-pick PR that puts this commit over in the "current" branch.

Thanks! Done in https://github.com/kiali/kiali.io/pull/561

**Pull request merged**